public/app/features/dashboard/dashgrid/PanelChrome.tsx

```diff
@@ -297,10 +308,13 @@ export class PanelChrome extends Component<Props, State> {
     });
     const panelOptions = panel.getOptions();
 
+    // Necessary? will only affect *new* subscriptions?
+    (this.state.context.eventBus as any).localOnly = dashboard.graphTooltip === 0;
```

Comment:
@torkelo was this added so when you change dashboard settings the panels update?  Would it be better to just have dashboard changes (in a different page!) reinitalize the dashboard?

Reply:
what do you mean re-initialize the dashboard? after a simple settings change trigger a full reload, that is very expensive (triggers full unmount, load process, and variable re-init). Panel and dashboard setting changes usually just trigger react re-renders (when props change) 

But yes this was to update the eventBus scoping when dashboard tooltip option changes

Reply:
Lets revist this once we see other ways the filter should be re-configured.  The filter is called for every mouse move, and the shared cursor setting is modified um... typically never, but for sure infrequently.